datafusion/common/src/scalar.rs

```diff
@@ -1038,6 +1054,7 @@ macro_rules! impl_op_arithmetic {
                 get_sign!($OPERATION),
                 true,
             )))),
+            // todo: Add Decimal256 support
```

Comment:
The error of running

```
create table t as values (arrow_cast(123, 'Decimal256(5,2)'));
select AVG(column1) from t;
```

in decimal.slt comes from lack of implementation of Decimal256 Scalar arithmetic op.

It seems to be not easy to finish right now. So I left it as todo here.